string map_id 
float32 height_agl
float32[] lng
float32[] lat
---
bool success
string message

